setup.cfg

```diff
@@ -34,22 +33,25 @@ scripts =
   bin/patch_content_types
   bin/use_external_components
 install_requires =
-    django>=3.2.0
-    django-filter>=2.0
+    django>=4.2.0
+    django-filter>=23.1,<=25.1
     django-solo
-    djangorestframework>=3.11.0
+    djangorestframework>=3.15.0
     djangorestframework_camel_case>=1.2.0
     django-rest-framework-condition
-    drf-yasg>=1.20.0
-    drf-nested-routers>=0.93.3
-    gemma-zds-client>=0.14.0
+    drf-yasg>=1.20.0 # TODO: remove usage?
+    drf-nested-routers>=0.94.1
     iso-639
     isodate
-    notifications-api-common>=0.2.2
+    notifications-api-common>=0.3.0
+    zgw-consumers>=0.35.1
+    zgw-consumers-oas
```

Comment:
Why do we need this new dependency? Now we will use client which doesn't request OAS right?

Reply:
@stevenbal I changed this requirement in an [separate PR](https://github.com/maykinmedia/commonground-api-common/pull/40/commits/b420445442379a51c5aa35bd5de70d2c8393115d) (which points to this branch)

Reply:
this mean that projects that make use of these OAS utils have to install `commonground-api-common[tests]`